src/apiserver/client.go

```diff
@@ -40,19 +40,23 @@ func (c clientImpl) GetNodeInfo(nodeName string) (*NodeInfo, error) {
 	}
 
 	return &NodeInfo{
-		NodeName:    node.ObjectMeta.Name,
-		Labels:      node.Labels,
-		Allocatable: node.Status.Allocatable,
-		Capacity:    node.Status.Capacity,
+		NodeName:      node.ObjectMeta.Name,
+		Labels:        node.Labels,
+		Allocatable:   node.Status.Allocatable,
+		Capacity:      node.Status.Capacity,
+		Conditions:    node.Status.Conditions,
+		Unschedulable: node.Spec.Unschedulable,
```

Comment:
Commit message talks about conditions, but we also start fetching unschedulable here.

Reply:
Nice point, I'll add it to the description.